### DISCUSSION TITLE
So....What is the latest version anyway?

### DISCUSSION THREAD

**Discussion by Node815** [dani-garcia/vaultwarden#4559] [Q&A] [answered]
In my docker-compose I have it set to the latest tag.  
On GH, it shows as 1.30.5 (as of this writing) for the newest release.
Peeking at Docker hub I see several versions, but it seems the one tagged for "latest"  is not showing a version number so it may be months older than this?  My version says " Version 2024.1.2" and works fine so I'm okay staying there, but if there is an update to be had.. :)

**Comment by BlackDex** [comment, marked as answer]
If you look at the date/time at docker hub you see it states 2 months
![image](https://github.com/dani-garcia/vaultwarden/assets/610450/cf0c5874-9283-44c0-b95b-9e524d2d4610)

Also, if you check the hashes of `latest` and `1.30.5` you would see they are the same.
![image](https://github.com/dani-garcia/vaultwarden/assets/610450/97beb3c8-ea0e-4a9e-82d9-5d12468ef7f5)

If you want the cutting-edge version, use the `testing` tag.
![image](https://github.com/dani-garcia/vaultwarden/assets/610450/3f848fb4-e4ef-491d-9b1e-0593ad5ab2f8)

**Comment by Node815** [comment]
Thanks!  It's all good, I was just going through my sweep of older installs and was checking here, Like I said it's been working fine. :)
